pkg/common/cns-lib/vsphere/datacenter.go

```diff
@@ -101,7 +101,8 @@ func (dc *Datacenter) GetVirtualMachineByUUID(ctx context.Context,
 	searchIndex := object.NewSearchIndex(dc.Datacenter.Client())
 	svm, err := searchIndex.FindByUuid(ctx, dc.Datacenter, uuid, true, &instanceUUID)
 	if err != nil {
-		log.Errorf("failed to find VM given uuid %s with err: %v", uuid, err)
+		log.Errorf("couldn't find VM for the given uuid %s Either VM is not present "+
```

Comment:
Minor update to the sentence here - "couldn't find VM for the given uuid %s. Either VM is not present "